src/Conf.ml

```diff
@@ -1513,9 +1523,18 @@ let parse_line config ~from s =
           ~from:(`Parsed (`File x))
           ~name ~value ~inline:false
     | name, `Attribute ->
-        C.update ~config
-          ~from:(`Parsed `Attribute)
-          ~name ~value ~inline:true
+        if
+          !disable_conf_attrs
+          && (not (String.equal name "disable"))
+          && not config.quiet
```

Comment:
If `--quiet` is passed, attributes are not ignored. You meant a nested `if` ?

Reply:
good catch